tck/src/main/resources/conf/jdori-log4j2.xml

```diff
@@ -38,7 +38,7 @@
     
     <!-- TCK Loggers -->
 
-    <Logger name="org.apache.jdo.tck" level="info" additivity="false">
+    <Logger name="org.apache.jdo.tck" level="debug" additivity="false">
```

Comment:
That looks like an unrelated change. Is it intended?

Reply:
Good catch! This change is not intended. I will revert this.